README.md

```diff
@@ -22,6 +22,12 @@ Having to maintain multiple versions of the language forever wouldn't actually m
 
 Browsers are [unwilling to break web pages](https://developer.chrome.com/blog/smooshgate/#why-dont-we-just-keep-the-existing-name-and-break-the-web) for their users, no matter how much you think the developers of those websites deserve it. However, "not breaking the web" does not mean 100% backward compatibility. Backward-incompatible changes *can* be made when the undesirable behaviour is not being relied upon, but the web is very large, which makes it both hard to confidently determine this and unlikely except for very minor or very strange behaviours.
 
+### Will JavaScript ever get a type system?
```

Comment:
```suggestion
### Will JavaScript ever support static typing?
```

Reply:
maybe:
```suggestion
### Will JavaScript ever get a static type system?
```
?

Reply:
@ljharb's suggestion is acceptable, but we should not elaborate any further.

Reply:
"static type system" SGTM. Alternatively we could use examples:

> Will JavaScript ever get a built-in type system similar to TypeScript or Flow?